app/src/app/list/polls-table/polls-table.component.sass

```diff
@@ -3,28 +3,38 @@
 @import '../../../sizes'
 
 table
+  display: grid
+  grid-template-columns: 1fr min-content min-content min-content min-content
   border: solid 1px $maincolor
-  border-collapse: collapse
-  thead
-    border: solid 1px $maincolor
-  td, th
+  tbody, tr, thead
+    display: contents
+  th
+    border-bottom: solid 1px $maincolor
+    background-color: $softbgcolor
     padding: 2px 4px
   th.manage
     padding: 2px 0
-  tr.tableheader
-    background-color: $softbgcolor
   tr.poll
+    &:hover td
+      background-color: $softbgcolor
```

Comment:
Monsieur After moving this rule to the general rule of the td elements, the changing of background just happens for each td, but not for each line tr...

Reply:
This is probably because you changed the selector. I do not ask you to change the selector, just to move the lines (without changing anything, not even the indentation).